popup.html

```
@@ -45,6 +45,11 @@ <h2 id="phrase-text"></h2>
             <div class="actions">
                 <button id="toggle-explanation" class="btn-primary">Se forklaring</button>
                 <a id="lookup-link" href="#" target="_blank" class="btn-secondary">Slå op</a>
+                <button id="save-btn" class="btn-save" title="Gem ord">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)